src/table/components/TableWrapper.jsx

```diff
@@ -113,33 +113,51 @@ export default function TableWrapper(props) {
     });
   }, [rows.length, size.qcy, size.qcx, page]);
 
-  const tableBackgroundColor = theme.getStyle('object', 'straightTable', 'backgroundColor');
+  const SelectProps = {
+    inputProps: {
+      'aria-label': translator.get('SNTable.Pagination.RowsPerPage'),
+      'data-testid': 'select',
+      tabIndex: !keyboard.enabled || keyboard.active ? 0 : -1,
+    },
+    native: true,
+  };
+
   const paperStyle = {
```

Comment:
I think all style objects should be defined in styling-utils. `getTableStyle` or something that returns an object with all the diffrent styles which you the destruct. As for the other files, create a useMemo that depends on theme and constraints. Would slim down this file, and make it easier to test

Reply:
Yup, I will add it to the issue and for future refactoring.